exotica_core/include/exotica_core/kinematic_tree.h

```diff
@@ -145,9 +154,16 @@ class KinematicTree : public Uncopyable
     const Eigen::MatrixXd& GetJointLimits() const { return joint_limits_; }
     void SetJointLimitsLower(Eigen::VectorXdRefConst lower_in);
     void SetJointLimitsUpper(Eigen::VectorXdRefConst upper_in);
+    void SetJointVelocityLimits(Eigen::VectorXdRefConst velocity_in);
+    void SetJointAccelerationLimits(Eigen::VectorXdRefConst acceleration_in);
+    void SetFloatingBaseLimitsPosXYZEulerZYX(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration);
+    void SetPlanarBaseLimitsPosXYEulerZ(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration);
```

Comment:
Why do we still need the overload without vel/acc?
The API will be preserved if you declare it as:
```
void SetFloatingBaseLimitsPosXYZEulerZYX(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity = {}, const std::vector<double>& acceleration = {});
```

Reply:
You originally told me to overload the function - I can not do that if you wish. I'll do the formatting as you suggest